gips/data/aod/aod.py

```diff
@@ -47,7 +51,7 @@ def data_path(cls, tile='', date=''):
 
     @classmethod
     def find_tiles(cls):
-        return ['']
+        return ['h01v01']
```

Comment:
Optional:  I found it handy in the sentinel-2 driver to set a module attribute `asset_type = 'L1C'`.  Then the compiler catches spelling errors for me and the error messages are obvious.  You could do that here if you want, call it `singleton_tid` or similar.